docs/debugging-testing/testing/unit_testing.md

```diff
@@ -12,11 +12,10 @@ Please install the following dependencies to use Mbed OS unit testing.
 
 - GNU toolchains.
    - GCC 6 or later. We recommend you use MinGW-W64 on Windows, but any Windows port of the above GCC versions works.
-- CMake 3.0 or newer.
+- CMake 3.19.0 or newer.
 - Python 2.7.x, 3.5 or newer.
 - Pip 10.0 or newer.
 - Gcovr 4.1 or newer.
-- Mbed CLI 1.8.0 or newer.
```

Comment:
I thought Mbed CLI  would still provide an interface for running unit test.

Reply:
Yes Mbed CLI 1 provide an interface using the `mbed test --unittests` command